config/manifests/bases/ssp-operator.clusterserviceversion.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       kind: SSP
       name: ssps.ssp.kubevirt.io
       version: v1beta2
-  description: KubeVirt Schedule, Scale and Performance Operator
+  description: Operator that deploying and controlling additional KubeVirt resources
```

Comment:
What about "Kubevirt Operator for deploying and controlling additional Kubevirt resources"

Reply:
`Operator that manages additional KubeVirt related resources`? Or at least `Operator that deploys and controls additional KubeVirt resources`